recipes/torchbiggraph/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "torchbiggraph" %}
+{% set version = "1.dev1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/facebookresearch/PyTorch-BigGraph/archive/v{{ version }}.tar.gz
+  sha256: 2e2a4809cf40c10aef35536f8dcabb9661eadf63b2b1479d765d5955faeb467f
+
+build:
+  number: 0
+  skip: True  # [py<36]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
```

Comment:
You can eliminate the build section; it is needed primarily for compiled packages.